resources/views/individuals/partials/about.blade.php

```diff
@@ -36,17 +36,21 @@
 
     <h4>{{ __('Groups in the disability and Deaf community') }}</h4>
```

Comment:
This heading should only appear when specific disability or deaf groups are indicated. See @cherylhjli's [response](https://matrix.to/#/!cTDzVKxAmJRmfbSdHL:matrix.org/$eGMLnzLjFRyYvsN2vag9X_ckdawcmzzNWY_sk0B8Bdg?via=matrix.org) in matrix.